lib/roo_on_rails/checks/github/branch_protection.rb

```diff
@@ -35,7 +36,9 @@ def fix
 
         def ensure_status_checks!
           status_checks = protection[:required_status_checks] || {}
-          fail! 'status checks do not include admins' unless status_checks[:include_admins]
+          enforce_admins = protection[:enforce_admins]
+          fail! 'status checks are not enforced for admins' unless enforce_admins &&
+                                                                   enforce_admins[:enabled]
```

Comment:
Weirdly in spite of the name it's actually a hash!

```
"enforce_admins": {
    "url": "https://api.github.com/repos/octocat/Hello-World/branches/master/protection/enforce_admins",
    "enabled": true
  },
```

([source](https://developer.github.com/v3/repos/branches/))